ui/apps/dashboard/graphql.config.yaml

```diff
@@ -33,6 +33,7 @@ extensions:
             NullTime: 'null | string'
             Period: 'unknown'
             Role: 'unknown'
+            RunStatus: 'unknown'
```

Comment:
Should `RunStatus` be mapped? Like we're doing in monorepo

Reply:
It seems to be duplicating [`FunctionRunStatus`](https://github.com/inngest/monorepo/blob/7795837e66fe1841b2162b22d986a79e688db856/api/schema.graphql#L754C1-L754C1). I added a comment about it on the other PR: https://github.com/inngest/monorepo/pull/2004/files#r1408342202